grouper/fe/templates/group.html

```diff
@@ -40,7 +40,7 @@
 
     <!-- Show the join button if the user isn't already a member and the group
          isn't invite-only. -->
-    {% if not current_user_role['is_approver'] and group.canjoin != "nobody" %}
+    {% if not current_user_role['is_member'] and group.canjoin != "nobody" %}
```

Comment:
tbh I feel Join should always be there -- maybe messaged differently? because what if I want to join another group to this group?